tests/core/factory/test_checkpoint_factory.py

```diff
@@ -221,6 +222,51 @@ def _test_checkpoint_factory_delete_success(context):
         context.checkpoints.get(name)
 
 
+@pytest.mark.parametrize(
+    "context_fixture_name",
+    [
+        pytest.param("empty_cloud_context_fluent", id="cloud", marks=pytest.mark.cloud),
+        pytest.param("ephemeral_data_context", id="ephemeral", marks=pytest.mark.unit),
```

Comment:
Is this a unit test? Maybe `big`?

Reply:
good call! will update

Reply:
fwiw I'm updating the corresponding test in each of these files. It's not technically a unit test, but I think it's also a pretty quick test, so I don't have strong opinions on the distinction 🤷